app/assets/javascripts/optional-checkbox-section.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+$(function () {
+
+  /**
+   * Links visibility of an optional section to the selection within a specified set of controls
+   * @param {jQuery wrapped input} $input - A radio button.
+   */
+
+  var manageStateOfOptionalCheckboxSection = function ($input, $optional_checkbox_section_wrapper) {
+    var id = $input.attr('id');
+    switch (true) {
+      case /_prison_staff$/.test(id):
```

Comment:
The repetition in this case statement isn't so good, it seems avoidable(check a list & perform an operation - all of the operations here are the same) but besides that it feels as though we need a way of being able to apply this without hardcoded field names. Is this code just copied from the other optional script?